### ISSUE TITLE
Target label choosing stragegies

### ISSUE THREAD

**Issue by GuanlinLee** [Wuziyi616/IF-Defense#6]
I notice that you use targeted attacks. However, the paper does not provide any details in target label choosing. As analysis in previous works, if you choose an easy case to build a target label, the accuracy under attack will be low.  So, could you provide more details in choosing target labels. If any possible, you can provide the pre-trained models. I want to test them under un-targeted attacks, which may show the robustness in another perspective.

**Comment by Wuziyi616** [comment]
Hi. Thank you for your interest in our work. We do provide pre-trained models in this repo, you can find it [here](https://drive.google.com/file/d/1n9bRWyjPWSMyQktodCP2fnKpmzDFEf3e/view). As for the target label generation, we just use random generation (which is also provided in our repo [here](https://drive.google.com/file/d/1o47ZvVcNvwBGv55xibEFw6KAOLaRF1IF/view)). Unfortunately, I didn't save the random seed I used for that. But I remember in our early stage of experiments, I tried different target labels and the attack results is similar. Maybe because 2468 is a large enough number to eliminate randomness.

Hope this can help you!

**Comment by GuanlinLee** [comment]
Yeah. Thanks for your explanation.
